group_vars/matrix_servers

```diff
@@ -705,11 +705,45 @@ matrix_mautrix_whatsapp_database_password: "{{ '%s' | format(matrix_homeserver_g
 
 ######################################################################
 #
-# matrix-bridge-mautrix-discord
+# matrix-bridge-mautrix-wsproxy
 #
 ######################################################################
 
 # We don't enable bridges by default.
+matrix_mautrix_wsproxy_enabled: false
+
+matrix_mautrix_wsproxy_systemd_required_services_list: |
+  {{
+    ['docker.service']
+    +
+    (['matrix-synapse.service'] if matrix_synapse_enabled else [])
+    +
+    (['matrix-postgres.service'] if matrix_postgres_enabled else []) 
+  }}
+
+matrix_mautrix_wsproxy_homeserver_domain: '{{ matrix_domain }}'
+
+matrix_mautrix_wsproxy_homeserver_address: "{{ 'http://matrix-synapse:8008' if matrix_synapse_enabled else '' }}"
+
+matrix_mautrix_wsproxy_homeserver_token: "{{ matrix_homeserver_generic_secret_key | password_hash('sha512', 'wsproxy.hs.token') | to_uuid }}"
+
+matrix_mautrix_wsproxy_appservice_token: "{{ matrix_homeserver_generic_secret_key | password_hash('sha512', 'wsproxy.as.token') | to_uuid }}"
+
+matrix_mautrix_wsproxy_login_shared_secret: "{{ matrix_synapse_ext_password_provider_shared_secret_auth_shared_secret if matrix_synapse_ext_password_provider_shared_secret_auth_enabled else '' }}"
```

Comment:
You have both `matrix_mautrix_wsproxy_syncproxy_shared_secret` and `matrix_mautrix_wsproxy_login_shared_secret`.

In fact, `matrix_mautrix_wsproxy_login_shared_secret` only appears here and doesn't seem to be used elsewhere.